Java/AutoML/automl-cms-api/src/main/java/com/automl/cms/api/model/Result.java

```diff
@@ -0,0 +1,70 @@
+package com.automl.cms.api.model;
+
+import static com.automl.component.constants.ResultCode.SUCCESS;
+import static com.automl.component.constants.ResultCode.UNKNOWN_ERROR;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author wangqingwei
+ * Created on 2021-02-20
+ */
+public class Result extends HashMap<String, Object> {
+    public Result() {
+        put("code", SUCCESS.getValue());
+    }
+
+    @SuppressWarnings("checkstyle:MagicNumber")
+    public static Result error() {
+        return error(UNKNOWN_ERROR.getValue(), "未知异常，请联系管理员");
+    }
+
+    @SuppressWarnings("checkstyle:MagicNumber")
+    public static Result error(String msg) {
+        return error(UNKNOWN_ERROR.getValue(), msg);
+    }
+
+    public static Result error(int code, String msg) {
+        Result r = new Result();
+        r.put("code", code);
+        r.put("msg", msg);
+        return r;
+    }
+
+    public static Result ok(Object data) {
```

Comment:
They are still here :)

Reply:
You mean we don't need the result class

Reply:
I mean to remove those functions called 'ok', they will not be used, right?

Reply:
yes，i will delete this class result,  it does't seem to useful anymore.